CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - **CUMULUS-2806**
     - Remove DynamoDB logic from executions `POST` endpoint
     - Remove DynamoDB logic from sf-event-sqs-to-db-records lambda execution writes.
+    **CUMULUS-2807**
+    - Remove DynamoDB logic from executions `PUT` endpoint
```

Comment:
May be a minor quibble, but given this was introduced in https://github.com/nasa/cumulus/pull/2967/files as part of the work in CUMULUS-2806 I believe this line entry should be added under CUMULUS-2806.